pixi/src/ui/recommendations/RecommendationItem.js

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import i18n from '../I18n.js';
+import {addTargetBlankToLinks, cleanCodeForInnerHtml} from '../../utils/texts';
+import marked from 'marked';
+
+export default class RecommendationItem {
+  constructor(content, pageUrl, element) {
+    this.content = content;
+    this.pageUrl = pageUrl;
+
+    this.element = element;
+    this.element.id = `recommendation-${content.id}`;
+    this.element.style = null;
+
+    this.header = element.querySelector(
+      '.ap-m-pixi-recommendations-item-header'
+    );
+    this.header.id = `header-${content.id}`;
+
+    this.title = element.querySelector(
+      '.ap-m-pixi-recommendations-item-header-title'
+    );
+    this.toggle = element.querySelector(
+      '.ap-m-pixi-recommendations-item-header-toggle'
+    );
+
+    this.body = element.querySelector('.ap-m-pixi-recommendations-item-body');
+    this.body.id = `body-${content.id}`;
+
+    this.bodyText = element.querySelector(
+      '.ap-m-pixi-recommendations-item-body-text'
+    );
+
+    this.nextButton = element.querySelector('a');
```

Comment:
Could this incorrectly get anchor links in side the recommendation body copy?

Reply:
Yep, true. Moved this above the rendering of the body.